client/lib/purchases/index.js

```diff
@@ -71,6 +73,15 @@ function getName( purchase ) {
 	return purchase.productName;
 }
 
+function getFancyName( purchase ) {
```

Comment:
I disagree with the naming choice here. What exactly does `fancy` mean? Can we use something more descriptive?

Reply:
Do you have any suggestions here for the name? I am happy to change it. 
The difference between the getFancyName and the getName is that in come cases. We italicize things. Which makes things fancy. I didn't want to be too descriptive since the fancyness might change due to the design requirements here.

Reply:
How about `getFormattedName` or `getRichName`? I think it describes the things a bit better while it keeps it flexible for any future changes.

Reply:
@tyxla I changed to getDisplayName() so that it is in line with the constant. 
What do you think?

Reply:
I love it! Thanks 👍